File: php/experiment/oo/testEmployee.php
<?php
// testEmployee.php

require_once("app_autoload.php");

$adam = new Employee("Adam", "Cameron");
echo "Name: " . $adam->getFullName() . "<br>";
echo "Sex: " . $adam->getSex() . "<br>";
echo "Population: " . Employee::getPopulation();
echo "<hr>";

require_once("../debug/dBug.php");
new dBug($adam);

File: php/www/experiment/oo/java/TestEmployee.java
// TestEmployee.java

public class TestEmployee {

	public static void main(String[] args){
		Person person = new Person("Zachary", "Daniel", "Cameron Lynch");
		System.out.println(person.getFullName());

		Employee employee = new Employee("Donald", "Adam", "Cameron", "17");
		System.out.println(employee.getFullName());
	} 
}